agenda.php

```diff
@@ -309,6 +316,89 @@ protected function getctag() {
         return NULL;
     }
     
+    //if add is true, then add $usermail to the event, otherwise, remove it.
+    function updateAttendee($url, $usermail, $add, $attendee_CN=NULL) {
+        $raw = file_get_contents_safe('./data/' . $url);
+        $etag = file_get_contents_safe('./data/' . $url . '.etag');
+        
+        $vcal = \Sabre\VObject\Reader::read($raw);
+        
+        if($add) {
+            $already_in = false;
```

Comment:
ah, it's cool that you though about handling this kind of corner case; good catch! 👍